src/util.rs

```diff
@@ -86,3 +86,18 @@ pub const fn weeks_in_year(year: i32) -> u8 {
         52
     }
 }
+
+pub(crate) trait Clamp {
```

Comment:
As this isn't used anywhere aside from the quickcheck module, this trait can be placed there (with private visibility).

Reply:
Done!